lib/table/features/date-edit/DateEdit.js

```diff
@@ -113,10 +115,15 @@ DateEdit.prototype.setupComplexCells = function() {
       }
 
       var node;
+      translate;  // sorry, i have to declare it here in order to pass "no-unused-vars" test..
       if (element.column.type === 'dmn:InputClause') {
-        node = domify(require('./template-input.html'));
+        node = domify(require('./template-input.html').replace(/\{\{(.+?)\}\}/g, function(_, match) {
+          return eval(match);
```

Comment:
Usage of `eval` is evil.

There should be another way to parse the templates, i.e. via a pattern and pipe it to translate.